package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "diaspora",
+  "version": "0.7.17.0",
+  "repository": "git@github.com:diaspora/diaspora.git",
+  "author": "diaspora* team",
+  "license": "AGPL-3.0",
```

Comment:
Can you add a

```
"private": true
```

as well? That's generally good practice for things that should not end up on npm.